<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tke_Form_uni_insert_kvytancii</class>
 <widget class="QWidget" name="tke_Form_uni_insert_kvytancii">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Втягування квитанцій (універсальна форма)</string>
  </property>
  <property name="windowIcon">
   <iconset resource="tke/tke_rc">
    <normaloff>:/pictures/pictures/cube.png</normaloff>:/pictures/pictures/cube.png</iconset>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QGroupBox" name="groupBox_chooseFile">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>3</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="title">
       <string>Вибір файлу квитанцій</string>
      </property>
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTreeView" name="treeView_fileMeneger"/>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_reloadDirTree">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Поновити дерево каталогів</string>
         </property>
         <property name="icon">
          <iconset resource="tke/tke_rc">
           <normaloff>:/pictures/pictures/redeal.png</normaloff>:/pictures/pictures/redeal.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>12</width>
           <height>12</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_reloadScript">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Поновити скрипт</string>
         </property>
         <property name="icon">
          <iconset resource="tke/tke_rc">
           <normaloff>:/pictures/pictures/redeal.png</normaloff>:/pictures/pictures/redeal.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>12</width>
           <height>12</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout">
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_banky">
           <property name="text">
            <string>Банки</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_vybirBanku">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox_kvyt">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>5</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Квитанції з файлу</string>
      </property>
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTabWidget" name="tabWidget_kvyt">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_kvyt">
          <attribute name="title">
           <string>Перелік квитанцій у файлі</string>
          </attribute>
          <layout class="QVBoxLayout">
           <property name="spacing">
            <number>2</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QTableWidget" name="tableWidget_kvytancii">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="sizeIncrement">
              <size>
               <width>50</width>
               <height>0</height>
              </size>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::SingleSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <column>
              <property name="text">
               <string>status</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Рахунок</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Дата оплати</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Сума, грн.</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Кас. апарат</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>№ пакету</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_KvytSum">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="Line" name="line">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout">
             <property name="spacing">
              <number>2</number>
             </property>
             <item>
              <widget class="QLabel" name="label_packNumber">
               <property name="text">
                <string>№ пакету</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox_packNumber">
               <property name="minimumSize">
                <size>
                 <width>60</width>
                 <height>0</height>
                </size>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="maximum">
                <number>100000</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="toolButton_setPackNumbers">
               <property name="text">
                <string>№ пакетів</string>
               </property>
               <property name="icon">
                <iconset resource="tke/tke_rc">
                 <normaloff>:/pictures/pictures/editors_section.png</normaloff>:/pictures/pictures/editors_section.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>12</width>
                 <height>12</height>
                </size>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonTextBesideIcon</enum>
               </property>
               <property name="autoRaise">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QToolButton" name="toolButton_addKvyt">
               <property name="text">
                <string>Додати квитанцію</string>
               </property>
               <property name="icon">
                <iconset resource="tke/tke_rc">
                 <normaloff>:/pictures/pictures/greplus.png</normaloff>:/pictures/pictures/greplus.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>12</width>
                 <height>12</height>
                </size>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonTextBesideIcon</enum>
               </property>
               <property name="autoRaise">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="toolButton_dellKvyt">
               <property name="text">
                <string>Видалити квитанцію</string>
               </property>
               <property name="icon">
                <iconset resource="tke/tke_rc">
                 <normaloff>:/pictures/pictures/redminus.png</normaloff>:/pictures/pictures/redminus.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>12</width>
                 <height>12</height>
                </size>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonTextBesideIcon</enum>
               </property>
               <property name="autoRaise">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QToolButton" name="toolButton_deleteNulRahKvyt">
               <property name="text">
                <string>Видалити квитанції з Рах=0</string>
               </property>
               <property name="icon">
                <iconset resource="tke/tke_rc">
                 <normaloff>:/pictures/pictures/redminus.png</normaloff>:/pictures/pictures/redminus.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>12</width>
                 <height>12</height>
                </size>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonTextBesideIcon</enum>
               </property>
               <property name="autoRaise">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="Line" name="line_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout">
             <property name="spacing">
              <number>2</number>
             </property>
             <item>
              <spacer>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QToolButton" name="toolButton_insertKvytToDB">
               <property name="text">
                <string>Внести квитанції до бази даних</string>
               </property>
               <property name="icon">
                <iconset resource="tke/tke_rc">
                 <normaloff>:/pictures/pictures/ark.png</normaloff>:/pictures/pictures/ark.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>12</width>
                 <height>12</height>
                </size>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonTextBesideIcon</enum>
               </property>
               <property name="autoRaise">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_packInfo">
          <attribute name="title">
           <string>Інформація про пакети</string>
          </attribute>
          <layout class="QVBoxLayout">
           <property name="spacing">
            <number>2</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QTextEdit" name="textEdit_packInfo"/>
           </item>
           <item>
            <layout class="QHBoxLayout">
             <property name="spacing">
              <number>2</number>
             </property>
             <item>
              <spacer>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QToolButton" name="toolButton_print">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Друк</string>
               </property>
               <property name="icon">
                <iconset resource="tke/tke_rc">
                 <normaloff>:/pictures/pictures/printer_default.png</normaloff>:/pictures/pictures/printer_default.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>12</width>
                 <height>12</height>
                </size>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonTextBesideIcon</enum>
               </property>
               <property name="autoRaise">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="toolButton_printOn">
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Друк на ...</string>
               </property>
               <property name="icon">
                <iconset resource="tke/tke_rc">
                 <normaloff>:/pictures/pictures/printer_conf.png</normaloff>:/pictures/pictures/printer_conf.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>12</width>
                 <height>12</height>
                </size>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonTextBesideIcon</enum>
               </property>
               <property name="autoRaise">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="tke/tke_rc"/>
 </resources>
 <connections/>
</ui>
